Production/govuk_ios/Extensions/UIKit/URLOpening.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+protocol URLOpening {
+    func canOpenURL(_ url: URL) -> Bool
+    func open(_ url: URL,
+              options: [UIApplication.OpenExternalURLOptionsKey: Any],
+              completionHandler completion: ((Bool) -> Void)?
+    )
```

Comment:
Can you move this `)` to line 7, please?

Reply:
Done